jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -211,12 +199,16 @@ async def job_script_list(
     "/job-scripts/{job_script_id}",
     status_code=status.HTTP_204_NO_CONTENT,
     description="Endpoint to delete job script",
-    dependencies=[Depends(guard.lockdown(Permissions.JOB_SCRIPTS_EDIT))],
 )
-async def job_script_delete(job_script_id: int = Query(..., description="id of the job script to delete")):
+async def job_script_delete(
+    job_script_id: int = Query(..., description="id of the job script to delete"),
+    token_payload: TokenPayload = Depends(guard.lockdown(Permissions.JOB_SCRIPTS_EDIT)),
+):
     """
     Delete job_script given its id.
     """
+    await _fetch_and_verify_ownership(job_script_id, token_payload)
```

Comment:
when looking at this request from swagger we will miss all the possible return codes and response. For example, the function `_fetch_and_verify_ownership` might return 400 or 403. This pattern is the same where this function is used across the code

Reply:
The Jobbergate API doesn't enumerate all possible error response codes for its endpoints. If you would like to see this functionality, please create a feature request for it. For this change, I will follow the established pattern for now.